libs/@blockprotocol/type-system/crate/src/ontology/url/mod.rs

```diff
@@ -114,6 +114,10 @@ impl FromStr for VersionedUrl {
     type Err = ParseVersionedUrlError;
 
     fn from_str(url: &str) -> Result<Self, ParseVersionedUrlError> {
+        if url.len() > 2048 {
```

Comment:
Can we add this to `BaseUrl`? As it was before

Reply:
It's still in `BaseUrl::validate_str`

Reply:
Ah apologies!